src/services/API/checkUser.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
```

Comment:
@deepansh96 can you guide here and help @sudheshna-donthineni extract all network calls to a service?